### PR TITLE
Enterprise 1.9: document ingress metrics data node config options

### LINKED ISSUE #2511: Document configuring and using ingress metrics
influxdata/influxdb#20613
influxdata/influxdb#20612

### PR DESCRIPTION
**Pull request by pierwill**
Closes #2511.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/administration/config-data-nodes.md [hunk 1]
```diff
@@ -228,6 +228,17 @@ InfluxData recommends values ranging from `0ms` to `100ms` for non-SSD disks.
 
 Environment variable: `INFLUXDB_DATA_WAL_FSYNC_DELAY`
 
+#### `ingress-metric-by-measurement-enabled = false`
+
+When `true`, collect statistics of points written per-measurement.
+These can be accessed via the monitoring subsystem.
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/config-data-nodes.md:234]: Is the "monitoring subsystem" on of the built-in buckets? Or is this something else? Would love to link the reader somewhere useful here...

Comment (lesam) [reply content/enterprise_influxdb/v1.9/administration/config-data-nodes.md:234]: This means you can see the metrics in `debug/vars`, and also in the `_internal` db if that is configured.

File: content/enterprise_influxdb/v1.9/administration/config-data-nodes.md [hunk 2]
```diff
@@ -228,6 +228,20 @@ InfluxData recommends values ranging from `0ms` to `100ms` for non-SSD disks.
...
+These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if that is configured.
+
+Environment variable: `INFLUXDB_DATA_INGRESS_METRIC_BY_MEASUREMENT_ENABLED`
+
+#### `ingress-metric-by-login-enabled = false`
+
+When `true`, collect statistics of points written per-login.
+These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if that is configured.
+
+Environment variable: `INFLUXDB_DATA_INGRESS_METRIC_BY_LOGIN_ENABLED`
```
Comment (pierwill) [inline content/enterprise_influxdb/v1.9/administration/config-data-nodes.md:243]: Are these environment variables correct, @lesam ?

Comment (lesam) [reply content/enterprise_influxdb/v1.9/administration/config-data-nodes.md:243]: Yes they are. We don't do the env variables individually, so as long as they follow the right pattern they will override the right piece of config.

File: content/enterprise_influxdb/v1.9/administration/config-data-nodes.md [hunk 3]
```diff
@@ -228,6 +228,20 @@ InfluxData recommends values ranging from `0ms` to `100ms` for non-SSD disks.
 
 Environment variable: `INFLUXDB_DATA_WAL_FSYNC_DELAY`
 
+#### `ingress-metric-by-measurement-enabled = false`
+
+When `true`, collect statistics of points written per-measurement.
+These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if that is configured.
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/config-data-nodes.md:234]:
```suggestion
These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if enabled.
```

File: content/enterprise_influxdb/v1.9/administration/config-data-nodes.md [hunk 4]
```diff
@@ -228,6 +228,20 @@ InfluxData recommends values ranging from `0ms` to `100ms` for non-SSD disks.
...
+
+Environment variable: `INFLUXDB_DATA_INGRESS_METRIC_BY_MEASUREMENT_ENABLED`
+
+#### `ingress-metric-by-login-enabled = false`
+
+When `true`, collect statistics of points written per-login.
+These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if that is configured.
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/administration/config-data-nodes.md:241]:
```suggestion
These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if enabled.
```

**Review by sanderson — APPROVED** [review]
LGTM. Two minor suggestions.

**Pull request merged**